applications/stream-applications-core/pom.xml

```diff
@@ -144,28 +144,37 @@
+                                    <management.prometheus.metrics.export.enabled>false</management.prometheus.metrics.export.enabled>
+                                    <management.prometheus.metrics.export.rsocket.enabled>false</management.prometheus.metrics.export.rsocket.enabled>
+                                    <!-- Add SCDF stream metrics tags. -->
                                     <management.metrics.tags.stream.name>"${spring.cloud.dataflow.stream.name:unknown}"</management.metrics.tags.stream.name>
                                     <management.metrics.tags.application.name>"${spring.cloud.dataflow.stream.app.label:unknown}"</management.metrics.tags.application.name>
                                     <management.metrics.tags.application.type>"${spring.cloud.dataflow.stream.app.type:unknown}"</management.metrics.tags.application.type>
                                     <management.metrics.tags.instance.index>"${spring.cloud.stream.instanceIndex:0}"</management.metrics.tags.instance.index>
                                     <management.metrics.tags.application.guid>"${spring.cloud.application.guid:unknown}"</management.metrics.tags.application.guid>
-
                                     <!-- Wavefront Tracing -->
                                     <wavefront.application.name>"${spring.cloud.dataflow.stream.name:unknown}</wavefront.application.name>
                                     <wavefront.application.service>${spring.cloud.dataflow.stream.app.label:unknown}-${spring.cloud.dataflow.stream.app.type:unknown}"</wavefront.application.service>
-
+                                    <!-- Zipkin Tracing -->
+                                    <management.tracing.enabled>false</management.tracing.enabled>
+                                    <management.tracing.sampling.probability>1.0</management.tracing.sampling.probability>
                                 </properties>
                                 <metadata>
                                     <mavenPluginVersion>${spring-cloud-dataflow-apps-metadata-plugin.version}</mavenPluginVersion>
                                 </metadata>
                                 <maven>
                                     <dependencyManagement>
+                                        <dependency>
+                                            <groupId>io.micrometer.prometheus</groupId>
+                                            <artifactId>prometheus-rsocket-spring</artifactId>
+                                            <version>${prometheus-rsocket.version}</version>
+                                        </dependency>
+                                        <dependency>
+                                            <groupId>io.micrometer.prometheus</groupId>
+                                            <artifactId>prometheus-rsocket-client</artifactId>
```

Comment:
Doesn't this come as a transitive dep of the previous one?

Reply:
Yep, good catch!